modules/bullet/space_bullet.h

```diff
@@ -124,6 +122,10 @@ class SpaceBullet : public RIDBullet {
 	real_t get_delta_time() { return delta_time; }
 	void step(real_t p_delta_time);
 
+	bool is_locked() const;
+	void lock();
+	void unlock();
```

Comment:
`lock()` and `unlock()` are never called, so better removing them, and `is_locked()` as well. They have a specific purpose in Godot Physics but I don't know if it makes sense with Bullet.